.gitignore

```diff
@@ -16,3 +16,18 @@ buildNumber.properties
 .idea
 .classpath
 *.iml
+
+### Maven ###
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+# https://github.com/takari/maven-wrapper#usage-without-binary-jar
+.mvn/wrapper/maven-wrapper.jar
```

Comment:
These lines are already present just above.
Except for `.mvn/wrapper/maven-wrapper.jar` but you can ignore it I don't use it.